examples/util/setupbitrategraph.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const DataSeries = require('./timelinegraph').DataSeries;
+const GraphView = require('./timelinegraph').GraphView;
+
+/**
+ * Set up the bitrate graph for audio or video media.
+ * @param {string} kind - 'video' or 'audio'.
+ * @param {string} containerId - The id of the graph container.
+ * @param {string} canvasId - The id of the canvas.
+ */
+ function setupBitrateGraph(kind, containerId, canvasId) {
+  const bitrateSeries = new DataSeries();
+  const bitrateGraph = new GraphView(containerId, canvasId);
+
+  bitrateGraph.graphDiv_.style.display = 'none';
+  return async function startBitrateGraph(room, intervalMs) {
+    let bytesReceivedPrev = 0;
+    let timestampPrev = Date.now();
+    const interval = setInterval(async function() {
```

Comment:
same thing here...the function passed to setInterval need not be async
```suggestion
    const interval = setInterval(function() {
```